.NET/Microsoft.Recognizers.Text.DateTime/DateTimeOptions.cs

```diff
@@ -38,6 +38,11 @@ public enum DateTimeOptions
         /// </summary>
         NoProtoCache = 16,
 
+        /// <summary>
+        /// NoProtoCache

```

Comment:
Please update comments

Reply:
I'd also personally prefer to use 2^20 as the value.

Reply:
Any specific reason for using 2^20 value.

Reply:
It's the highest available value.

Reply:
The file doesn't seem updated here.